### PR TITLE
NODE-5583 Required variables for smoke-tests

### PR DESCRIPTION
**Pull request by alex-drongo**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,18 +19,6 @@ IMAGE_LATEST := $(REGISTRY)/node:latest
...
 NODE_VERSION    ?= $(shell echo $(AIO_VERSION) | awk -F'[-.]' '{print $$1"."$$2"."$$3}')
 
-### Variables required to run test
-.EXPORT_ALL_VARIABLES:
-ALLURE_ENDPOINT       ?= https://allure.wallarm.com
-ALLURE_PROJECT_ID     ?= 10
-WALLARM_API_HOST      ?= api.wallarm.com
-WALLARM_API_CA_VERIFY ?= True
-CLIENT_ID             ?= 5
-PYTEST_WORKERS        ?= 10
```
Comment (xDmitriev) [inline Makefile:29]: Keep pytest_workers and pytest_args defaults here, they are not required

File: test/smoke_test.sh [hunk 2]
```diff
@@ -23,10 +23,81 @@ function get_logs_clean_and_exit() {
...
+    if [[ "${CI:-false}" == "true" ]]; then
+      mandatory=(
+        WALLARM_API_TOKEN
+        WALLARM_API_HOST
+        WALLARM_API_PRESET
+        CLIENT_ID
+        USER_TOKEN
+        WEBHOOK_API_KEY
+        WEBHOOK_UUID
+        PYTEST_WORKERS
```
Comment (xDmitriev) [inline test/smoke_test.sh]: Why are pytest_* variables mandatory?

File: test/smoke_test.sh [hunk 3]
```diff
@@ -23,10 +23,81 @@ function get_logs_clean_and_exit() {
...
+        PYTEST_ARGS
+      )
+    else
+      mandatory=(
+        WALLARM_API_TOKEN
+        WALLARM_API_HOST
+        WALLARM_API_PRESET
+        CLIENT_ID
+        USER_TOKEN
+        SMOKE_REGISTRY_TOKEN
```
Comment (xDmitriev) [inline test/smoke_test.sh]: Can we avoid such duplication and check registry creds variables separately? Same as we check allure variables

File: .github/workflows/ci.yaml [hunk 4]
```diff
@@ -46,7 +46,6 @@ jobs:
     env:
       ## US preset
```
Comment (xDmitriev) [inline .github/workflows/ci.yaml]: Please keep client ID here, it's part of presets

**Comment by xDmitriev** [comment]
Please make everything in 1-2 commits instead of 6

File: test/smoke_test.sh (hunk 3 above)
Comment (alex-drongo) [reply test/smoke_test.sh]: yes we can.

File: test/smoke_test.sh (hunk 2 above)
Comment (alex-drongo) [reply test/smoke_test.sh]: deleted from mandatory list

File: .github/workflows/ci.yaml (hunk 4 above)
Comment (alex-drongo) [reply .github/workflows/ci.yaml]: moved

File: Makefile (hunk 1 above)
Comment (alex-drongo) [reply Makefile:29]: moved

**Review by xDmitriev — APPROVED** [review]

**Pull request merged**